runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java

```diff
@@ -157,6 +160,18 @@ protected void shutDown() throws Exception
 		stickTile = null;
 	}
 
+	@Subscribe
+	public void onOverlayMenuClicked(OverlayMenuClicked overlayMenuClicked)
+	{
+		OverlayMenuEntry overlayMenuEntry = overlayMenuClicked.getEntry();
+		if (overlayMenuEntry.getMenuAction() == MenuAction.RUNELITE_OVERLAY
+				&& overlayMenuClicked.getEntry().getOption().equals(LapCounterOverlay.AGILITY_RESET)
+				&& overlayMenuClicked.getOverlay() == lapCounterOverlay)
```

Comment:
These indents are one level too far.
```suggestion
			&& overlayMenuClicked.getEntry().getOption().equals(LapCounterOverlay.AGILITY_RESET)
			&& overlayMenuClicked.getOverlay() == lapCounterOverlay)
```

Reply:
Also, checking overlay equality is cheaper than doing a string comparison. Switch the order of the overlay check and the option check.